CORDOVA_GUIDE.md

```diff
@@ -0,0 +1,430 @@
+## Installation - Branch PhoneGap or Cordova SDK
+
+### Available in Cordova
+
+Branch is available through [Cordova](http://plugins.cordova.io/#/package/io.branchmetrics.branchreferral), to install it simply execute the following line in terminal, while in your project directory:
+
+    cordova plugin add io.branchmetrics.branchreferral
+
+### Register you app
+
+You can sign up for your own API key at [https://dashboard.branch.io](https://dashboard.branch.io)
+
+## Configuration (for tracking)
+
+Ideally, you want to use our links any time you have an external link pointing to your app (share, invite, referral, etc) because:
+
+1. Our dashboard can tell you where your installs are coming from
+1. Our links are the highest possible converting channel to new downloads and users
+1. You can pass that shared data across install to give new users a custom welcome or show them the content they expect to see
+
+Our linking infrastructure will support anything you want to build. If it doesn't, we'll fix it so that it does: just reach out to alex@branch.io with requests.
+
+### Initialize SDK And Register Deep Link Routing Function
+
+Called when an app first initializes a session, ideally in the app delegate. If you created a custom link with your own custom dictionary data, you probably want to know when the user session init finishes, so you can check that data. Think of this callback as your "deep link router." If your app opens with some data, you want to route the user depending on the data you passed in. Otherwise, send them to a generic install flow.
+
+This deep link routing callback is called 100% of the time on init, with your link params or an empty dictionary if none present.
+
+**[Formerly `getInstance() and initSession()`](CORDOVA_UPGRADE_GUIDE.md)**
+
+
+```js
+// adds an instance of branch to the window object
+(function(b,r,a,n,c,h,_,s,d,k){if(!b[n]||!b[n]._q){for(;s<_.length;)c(h,_[s++]);d=r.createElement(a);d.async=1;d.src="https://cdn.branch.io/branch-v1.3.4.min.js";k=r.getElementsByTagName(a)[0];k.parentNode.insertBefore(d,k);b[n]=h}})(window,document,"script","branch",function(b,r){b[r]=function(){b._q.push([r,arguments])}},{_q:[],_v:1},"init data setIdentity logout track link sendSMS referrals credits redeem banner closeBanner".split(" "),0);
```

Comment:
fyi - for cordova, due to some stupidity, it's always going to be there, and always global - no need to put the snippet in